fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -7043,12 +7036,12 @@ public void replayBackendTabletsInfo(BackendTabletsInfo backendTabletsInfo) {
     public void convertDistributionType(Database db, OlapTable tbl) throws DdlException {
         tbl.writeLockOrDdlException();
         try {
-            if (!tbl.convertRandomDistributionToHashDistribution()) {
-                throw new DdlException("Table " + tbl.getName() + " is not random distributed");
+            if (!tbl.convertHashDistributionToRandomDistribution()) {
```

Comment:
Could you add some unit tests for your changes?

Reply:
done